src/config/i18n.ts

```diff
@@ -120,6 +120,7 @@ export const localeDirectoryMap = new Map<string, string>([
   ['ja', 'ja'],
   ['nl', 'nl'],
   ['pl', 'pl'],
+  ['pt-br', 'pt-br'],
```

Comment:
This should be:

```suggestion
  ['pt-BR', 'pt-BR'],
```

because the values are used in [getBrowserLocale](https://github.com/codeaid/woth-toolbox/blob/master/src/lib/i18n.ts#L180-L188), which in turn uses [get-browser-locale](https://www.npmjs.com/package/get-user-locale) to read configured user locales in the browser and the code for Brazilian Portugese is `pt-BR`.

With values being lowercase `if (localeDirectoryMap.has(locale))` won't trigger.